tests/kola/raid/config.ign

```diff
@@ -0,0 +1,88 @@
+{
+  "ignition": {
+    "version": "3.0.0"
+  },
+  "storage": {
+    "disks": [
+      {
+        "device": "/dev/disk/by-id/virtio-disk1",
+        "partitions": [
+          {
+            "label": "foo"
+          }
+        ],
+        "wipeTable": true
+      },
+      {
+        "device": "/dev/disk/by-id/virtio-disk2",
+        "partitions": [
+          {
+            "label": "bar"
+          }
+        ],
+        "wipeTable": true
+      },
+      {
+        "device": "/dev/disk/by-id/virtio-disk3",
+        "partitions": [
+          {
+            "label": "baz"
+          }
+        ],
+        "wipeTable": true
+      },
+      {
+        "device": "/dev/disk/by-id/virtio-disk4",
+        "partitions": [
+          {
+            "label": "boo"
+          }
+        ],
+        "wipeTable": true
+      }
+    ],
+    "raid": [
+      {
+        "devices": [
+          "/dev/disk/by-partlabel/foo",
+          "/dev/disk/by-partlabel/bar"
+        ],
+        "level": "raid1",
+        "name": "foobar"
+      },
+      {
+        "devices": [
+          "/dev/disk/by-partlabel/baz",
+          "/dev/disk/by-partlabel/boo"
+        ],
+        "level": "raid1",
+        "name": "bazboo"
+      },
+      {
```

Comment:
The test could be improved by listing this object first, so in-order assembly will fail.